src/client/pmix_client_get.c

```diff
@@ -186,12 +187,14 @@ static void _value_cbfunc(pmix_status_t status, pmix_value_t *kv, void *cbdata)
     pmix_cb_t *cb = (pmix_cb_t*)cbdata;
     pmix_status_t rc;
 
+    PMIX_ACQUIRE_OBJECT(cb);
```

Comment:
I don't think this Acquire is required since it's called from `_getnbfn`, but it's not hurting anything and I suppose it protects us in the future if we call this from another location.